variables.tf

```diff
@@ -90,6 +90,18 @@ variable "scan_host_vulnerabilities" {
   default     = true
 }
 
+variable "scan_multi_volume" {
+  type = bool
+  description = "Whether to scan secondary volumes. Defaults to `false`."
+  default = false
+}
+
+variable "scan_multi_volume" {
```

Comment:
Copy&paste error? `scan_stopped_instances`

Reply:
Indeed! Thank you